osflow/build_arty.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+
+export XRAY_DIR=~/osflow/prjxray_git
+export NEXTPNR_DIR=~/osflow/nextpnr-xilinx
+
+source "${XRAY_DIR}/utils/environment.sh"
+
+make BOARD=nexys_a7 MinimalBoot GHDL_PLUGIN_MODULE=ghdl
+
+openFPGALoader -b arty ./neorv32_nexys_a7_MinimalBoot.bit
```

Comment:
This should be added to the partial makefile in `osflow/boards/`. See, for instance, https://github.com/stnolting/neorv32-setups/blob/a087a44c8e11588dcff4bc44e922dbf4b80ac3ac/osflow/boards/Fomu.mk#L13-L14

Reply:
makes sense